wdio.helpers.d/assertions.js

```diff
@@ -0,0 +1,50 @@
+const addCommand = require('./lib/add-command');
```

Comment:
These helpers would be useful in the react repo too. Should we make them a package or put them in the web styleguide repo?

Reply:
probably, but we really need a template for new standalone projects with semantic-release fully automated.

Reply:
we could put them in the styleguide, but then we'd probably need to justify it as a "style rule" somehow, though there is something to be said for disallowing `browser.waitUntil` in spec files. That still runs into the issue of not having a fully automated release process.